### PR TITLE
Fixed instance_notifier to work with new PreGA

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Changed instance notifier to work with pre_GA instead of Master.

### CODE REVIEW

File: Tests/instance_notifier.py [hunk 1]
```diff
@@ -131,14 +131,11 @@ def slack_notifier(slack_token, secret_conf_path, server, user, password, build_
     if options.instance_tests:
         with open('./env_results.json', 'r') as json_file:
             env_results = json.load(json_file)
-            for env in env_results:
-                if env["Role"] == "Server Master":
-                    server = SERVER_URL.format(env["InstanceDNS"])
-                    break
+            server = SERVER_URL.format(env_results[0]["InstanceDNS"])
```
Comment (yaakovi) [inline Tests/instance_notifier.py]: not sure if you will get a list in case there is only one instance

File: .circleci/config.yml [hunk 2]
```diff
@@ -277,9 +277,12 @@ jobs:
             fi
             if ./Tests/scripts/is_ami.sh ;
               then
-                if [ -n "${NIGHTLY}" ] || [ -n "${INSTANCE_TESTS}" ] ;
+                if [ -n "${NIGHTLY}" ] ;
                   then
                     export IFRA_ENV_TYPE=Nightly # disable-secrets-detection
+                elif [ -n "${INSTANCE_TESTS}" ] ;
+                  then
+                    export IFRA_ENV_TYPE=Demisto PreGA # disable-secrets-detection
```
Comment (yaakovi) [inline .circleci/config.yml]: better safe than sorry
```suggestion
                    export IFRA_ENV_TYPE="Demisto PreGA" # disable-secrets-detection
```

File: Tests/instance_notifier.py (hunk 1 above)
Comment (ShahafBenYakir) [reply Tests/instance_notifier.py]: I think I will, but in any case, will run an instance build on this branch.

File: .circleci/config.yml (hunk 2 above)
Comment (ShahafBenYakir) [reply .circleci/config.yml]: :yaakovi:

**Pull request merged**